fishfish/__init__.py

```diff
@@ -15,5 +15,5 @@
 
 from .enums import Category, Permission
 from .models import URL, APIStatus, Domain, Token
-
-print("FISHFISH: THIS LIBRARY HAS NOT YET BEEN RELEASED")
+from .exceptions import FishFishException, Forbidden, Unauthorized, ServerError
```

Comment:
I wonder why the exception `ObjectDoesntExist` and the exception `Authenticated` aren't imported here.

Reply:
That'll be due to me being used to * stars for my exception files so I forgot